databases/de-database-schema/src/main/conversions/v2.8.0/c280_2016071201.clj

```diff
@@ -0,0 +1,39 @@
+(ns facepalm.c280-2016071201
+  (:use [kameleon.sql-reader :only [exec-sql-statement load-sql-file]])
+  (:require [korma.core :as sql]))
+
+(def ^:private version
+  "The destination database version"
+  "2.8.0:20160712.01")
+
+(defn- add-user-id-column
+  "Adds a user_id column to the integration_data table."
+  []
+  (println "\t* adding the user_id column to the integration_data table.")
+  (->> ["ALTER TABLE ONLY integration_data ADD COLUMN user_id uuid"
+        "ALTER TABLE ONLY integration_data ADD CONSTRAINT integration_data_user_id_fk
+         FOREIGN KEY (user_id) REFERENCES users (id)"]
```

Comment:
Minor suggestion: I would have loaded the constraint with `(load-sql-file "constraints/77_integration_data.sql")`, since the constraint is already written there for db inits (DRY), plus loading it this way kind of validates the SQL in that file without testing a full db init.